data/saas/config/mailchimp_config.yml

```diff
@@ -11,15 +11,12 @@ saas_config:
 
   client_config:
     protocol: https
-    host:
-      connector_param: domain
+    host: <domain>
     authentication:
-      strategy: basic_authentication
```

Comment:
I thought it was redundant to specify "authentication" in the strategy name so I made the following changes:

- `basic_authentication` -> `basic`
- `bearer_authentication` -> `bearer`